dodo.py

```diff
@@ -63,11 +64,13 @@ def task_cleanup():
 
     Usage: doit cleanup:<project>
     """
+    template_name=sys.argv[1].split(':')[1]
```

Comment:
If the project name is left out (ie meaning we want to run the command against all projects), then line 67 throws an exception since `1` is our of range of the list.